File-Formats/DocIO/Working-with-Tables.md

```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(imageStream);
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word file to MemoryStream
+    MemoryStream stream = new MemoryStream();
+    await document.SaveAsync(stream, FormatType.Docx);
+    //Saves the stream as Word file in local machine
+    Save(stream, "Result.docx");
+    //Refer to the following link to save Word document in UWP platform
+    //https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Creates an instance of WordDocument class.
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
+    FileStream imageStream = new FileStream(@"Image.png", FileMode.Open, FileAccess.ReadWrite);
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(imageStream);
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word document to MemoryStream.
+    MemoryStream stream = new MemoryStream();
+    document.Save(stream, FormatType.Docx);
+    stream.Position = 0;
+    //Download Word document in the browser.
+    return File(stream, "application/msword", "Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Creates an instance of WordDocument class.
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
+    Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+    Stream imageStream = assembly.GetManifestResourceStream("GettingStarted.Assets.Image.png");
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(imageStream);
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word document to MemoryStream
+    MemoryStream stream = new MemoryStream();
+    document.Save(stream, FormatType.Docx);
+    //Save the stream as a file in the device and invoke it for viewing
+    Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Result.docx", "application/msword", stream);
+    //Download the helper files from the following link to save the stream as file and open the file for viewing in Xamarin platform
```

Comment:
//Download the helper files from the following link to save the stream as a file and open the file for viewing on the Xamarin platform: